### PR TITLE
LT tutorial [WIP]

### PR DESCRIPTION
**Pull request by Krovatkin**
Fixes #ISSUE_NUMBER

### CODE REVIEW

File: torch/csrc/lazy/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+```
+maybe_true_lazy = torch.BoolTensor([1]).to(dev)
+lazy_result = add_two_maybe(t_lazy, maybe_true_lazy)
+assert(lazy_result.cpu(), add_two_maybe(t, maybe_true))
+```
+
+Woohooo! This works too!
+Unfortunately, this flexibility comes with a few downsides. Remember that backends need to translate aten ops into some much lower-level operations that an accelerator understands. The translation process may be time-consuming. Although, usually, it's well worth it! 
+
+TODO: Spiel about awesomeness of whole graph compilations
```
Comment (wconstab) [inline torch/csrc/lazy/tutorial.md:106]: lol.  really? maybe we don't need that here, i think it digresses the main point

Comment (Krovatkin) [reply torch/csrc/lazy/tutorial.md:106]: no spiel 😢

File: torch/csrc/lazy/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+lazy_result = add_two_maybe(t_lazy, maybe_true_lazy)
+assert(lazy_result.cpu(), add_two_maybe(t, maybe_true))
+```
+
+Woohooo! This works too!
+Unfortunately, this flexibility comes with a few downsides. Remember that backends need to translate aten ops into some much lower-level operations that an accelerator understands. The translation process may be time-consuming. Although, usually, it's well worth it! 
+
+TODO: Spiel about awesomeness of whole graph compilations 
+
+However, if you model is wildly dynamic and contains loops that always run different number of times or if statements one after another that explode into different traces every time you run your model, the backend will spend non-trivial amount of time compiling each trace even though the latter is used only for a few times.
```
Comment (wconstab) [inline torch/csrc/lazy/tutorial.md:108]: if your

a non-trivial

File: torch/csrc/lazy/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+if __name__='__main__':
+    bsz = 64
+    device = 'lazy'
+    epochs = 14
+    log_interval = 10
+    train_kwargs = {'batch_size': args.batch_size}
+    test_kwargs = {'batch_size': args.test_batch_size}
+    # if we want to use CUDA
+    if "LTC_TS_CUDA" in os.environ:
+        train_kwargs = {'num_workers': 1,
```
Comment (wconstab) [inline torch/csrc/lazy/tutorial.md:178]: cuda_kwargs?

**Review by wconstab — APPROVED** [review]
this is great! thanks so much @Krovatkin

File: torch/csrc/lazy/tutorial.md (hunk 3 above)
Comment (Krovatkin) [reply torch/csrc/lazy/tutorial.md:178]: Yup, last night I made sure the model runs and trains, I'll push the fixes shortly

**Comment by suo** [comment]
@Krovatkin hey in the future, do you mind editing the commit message with something more descriptive when landing? It helps people who are monitoring master understand the changes. Thanks :)

**Comment by malfet** [comment]
@Krovatkin why did you merge an issue with Work-in-progress in title? Als, please remove "Fixes #ISSUE_NUMER" if there are no issue associated with PR

**Pull request closed without merging**